eng/tools/generator/cmd/v2/common/fileProcessor.go

```diff
@@ -566,11 +565,25 @@ func replaceModuleImport(path, rpName, namespaceName, previousVersion, currentVe
 				}
 			}
 		}
-
 		return nil
 	})
 }
 
+func getModuleVersion(autorestPath string) (*semver.Version, error) {
+	data, err := os.ReadFile(autorestPath)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, line := range strings.Split(string(data), "\n") {
+		if strings.HasPrefix(line, autorest_md_module_version_prefix) {
+			return semver.NewVersion(strings.TrimSpace(line[len(autorest_md_module_version_prefix):]))
```

Comment:
Will `NewVersion` return an error when the value is not a valid version?

Reply:
yes

Reply:
What I meant is if NewVersion returns error on an invalid value then should we customize the error message before returning?

Reply:
There will be no invalid versions in autorest.md, and the program should be stopped if the module-version in autorest.md is invalid